src/app/themes.tsx

```diff
@@ -57,6 +61,8 @@ interface Props {
     theme?: CustomTheme;
 }
 
+let forceRerenderBecauseImTotallySane = 0;
```

Comment:
lol does this actually do anything? is there an implicit bug that needs to be fixed?